flags/flags.go

```diff
@@ -23,8 +23,8 @@ func Parse() Flags {
 	flag.BoolVar(&flags.Verbose, "v", false, "verbose output")
 	flag.BoolVar(&flags.JSONOutput, "j", false, "json output")
 	flag.BoolVar(&flags.SkipTLS, "insecure", false, "skip TLS verification")
-	flag.DurationVar(&flags.Interval, "interval", 5*time.Minute, "detection refresh interval")
-	flag.StringVar(&flags.Server, "server", "", "Run as http server instead of CLI. Output is exposed as JSON at /output. Default refersh interval is 5 minutes.")
+	flag.DurationVar(&flags.Interval, "interval", 5*time.Minute, "detection refresh interval when running in HTTP server mode")
+	flag.StringVar(&flags.Server, "server", "", "Run as HTTP server instead of CLI. Value must be a listen address (e.g.: 0.0.0.0:5000. Output is exposed as JSON at /output.")
```

Comment:
Only one thing left:

```suggestion
	flag.StringVar(&flags.Server, "server", "", "Run as HTTP server instead of CLI. Value must be a listen address (e.g.: 0.0.0.0:5000. Output is exposed as JSON at /detections.")
```